app/assets/javascripts/admin/addon/controllers/admin-badges-show.js

```diff
@@ -1,12 +1,12 @@
-import I18n from "I18n";
-import discourseComputed, { observes } from "discourse-common/utils/decorators";
-import { reads } from "@ember/object/computed";
 import Controller, { inject as controller } from "@ember/controller";
-import { popupAjaxError } from "discourse/lib/ajax-error";
+import discourseComputed, { observes } from "discourse-common/utils/decorators";
+import I18n from "I18n";
+import bootbox from "bootbox";
 import { bufferedProperty } from "discourse/mixins/buffered-content";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 import { propertyNotEqual } from "discourse/lib/computed";
+import { reads } from "@ember/object/computed";
 import { run } from "@ember/runloop";
-import bootbox from "bootbox";
```

Comment:
Not sure what happened here? Looks like thee imports got reordered when compared to core? Would be good to try and remove this diff.